settings.gradle

```diff
@@ -8,6 +8,7 @@ rootProject.name = 'opensearch-sql'
 
 include 'opensearch-sql-plugin'
 project(':opensearch-sql-plugin').projectDir = file('plugin')
+include 'api'
```

Comment:
Should we include it as part of the plugin snapshot and keep the plugin as a multipurposed jar like this? Seems like something where we should make the core SQL functionality a library, the opensearch plugin is one target that we can build, and the programmatic API is a different target we can build.

Reply:
Right, for now it is for external consumer. Later we can refactor `plugin` module to use this unified API as well.